cl/corpus_importer/management/commands/make_aws_manifest_files.py

```diff
@@ -48,8 +48,15 @@ def get_total_number_of_records(type: str, options: dict[str, Any]) -> int:
             WHERE is_available=True AND page_count>0 AND ocr_status!=1
             """
         case SEARCH_TYPES.OPINION:
-            base_query = "SELECT count(*) AS exact_count FROM search_opinion"
-            filter_clause = "WHERE extracted_by_ocr != true"
+            base_query = (
+                "SELECT count(*) AS exact_count "
+                "FROM search_opinion O "
+                "INNER JOIN search_opinioncluster OC ON O.cluster_id = OC.id"
+            )
+            filter_clause = (
+                "WHERE (extracted_by_ocr != true OR OC.source LIKE %s)"
+            )
+            params.append("'%U%'")
```

Comment:
I'm rusty on my postgres, but are LIKE queries slower than IN queries? We could do `"OC.source in %s" % ", ".join(HARVARD_SOURCES)` or something like that.

Reply:
Thanks for bringing this up. I ran `EXPLAIN ANALYZE` to compare the query plans and timing for both versions.

- Using LIKE
   
   Planning Time: 2,683 ms
   Execution Time: 41942,899 ms

- Using IN
   
   Planning Time: 2,483 ms
   Execution Time: 42925,716 ms

The query plans are identical in terms of the number of operations.  The only difference is the filter condition. The LIKE version completed execution slightly faster